protocol-units/mempool/util/src/lib.rs

```diff
@@ -147,12 +147,12 @@ impl PartialOrd for MempoolTransaction {
 impl Ord for MempoolTransaction {
 	fn cmp(&self, other: &Self) -> Ordering {
 		// First, compare by slot_seconds
-		match self.slot_seconds.cmp(&other.slot_seconds) {
```

Comment:
Good!
Is the `slot_seconds` field left with any use whatsoever now?

Reply:
Only insofar as we might want to make these structs self-describing.

Reply:
OK, I missed `Debug`. Is it just for that?

Reply:
I wasn't thinking about it just in terms of debug.